app/policies/proposal_policy.rb

```diff
@@ -98,7 +98,7 @@ def pending_delegate?
 
   def pending_approval!
     check(self.pending_approver? || self.pending_delegate?,
-          "A response has already been logged a response for this proposal")
+          "A response has already been logged for this proposal")
```

Comment:
sure, although I am not sure how we would get direct user feedback about this type of thing -- perhaps ping Amber and ask her to include it in her user interviews?